src/main/java/org/kiwiproject/dropwizard/util/exception/LoggingExceptionMapper.java

```diff
@@ -0,0 +1,86 @@
+package org.kiwiproject.dropwizard.util.exception;
+
+import static java.lang.String.format;
+
+import com.google.common.annotations.VisibleForTesting;
+import lombok.extern.slf4j.Slf4j;
+import org.kiwiproject.jaxrs.exception.JaxrsBadRequestException;
+import org.kiwiproject.jaxrs.exception.JaxrsException;
+import org.kiwiproject.jaxrs.exception.JaxrsExceptionMapper;
+import org.kiwiproject.jaxrs.exception.WebApplicationExceptionMapper;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Override default Dropwizard mapper to use common error structure. Catches all exceptions that don't have an explicit
```

Comment:
This should include the same verbiage as the other mappers about using `ErrorMessage` and `buildResponseEntity` etc.